.travis.yml

```diff
@@ -19,8 +19,10 @@ before_script:
   - sh -e /etc/init.d/xvfb start
   - npm install -g npm
   - npm install
+env:
+  - MIX_ENV=test
 script:
-  - "mix do local.hex --force, deps.get && mix test --include pandoc"
+  - "mix do local.hex --force, deps.get, test --include pandoc, coveralls.travis"
```

Comment:
This is not going to work because the `mix test` will complain it runs on the wrong environment. Keep the && and do `mix do` twice.